eng/pipelines/coreclr/templates/helix-queues-setup.yml

```diff
@@ -97,7 +97,7 @@ jobs:
         - (Debian.10.Amd64)Ubuntu.1804.amd64@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-10-helix-amd64
         - (Debian.11.Amd64)Ubuntu.1804.amd64@mcr.microsoft.com/dotnet-buildtools/prereqs:debian-11-helix-amd64
         - Ubuntu.1804.Amd64
-        - (Centos.8.Amd64)Ubuntu.1604.amd64@mcr.microsoft.com/dotnet-buildtools/prereqs:centos-8-helix
+        - (Centos.8.Amd64)Ubuntu.1804.amd64@mcr.microsoft.com/dotnet-buildtools/prereqs:centos-8-helix
```

Comment:
While I don't have a strong opinion about this change - the kernel in 18.04 is much newer and we'd lose some back coverage for test matrix of what's supported. Helix shows the queue existing. Do you have a link to the build?

Reply:
The real problem is centos8 is EOL. We haven't tested it in any significant manner. This needs to be closer to:

https://github.com/dotnet/runtime/blob/29d485c14f1ce5a7358cbed3dd97df725a5bec6d/eng/pipelines/coreclr/templates/helix-queues-setup.yml#L105-L109

It's not great we aren't testing ~16.04 kernels IMO, but at the very least I am OK with removing a couple of these queues. Also, we haven't tested CI in internal for a long time - only PR builds. So they should somewhat mimic PRs on main to have a baseline comparison. This applies for all the entries in this yml - Jeremy had updates this in main, but we never fixed servicing.